test/index.spec.js

```diff
@@ -281,6 +282,36 @@ describe('desciple-core-api', () => {
       expect(exportedData[ssid.did][1]).to.deep.equal({ [claimlink2]: { 'need': 'wine' } })
       expect(exportedData[ssid.did][2]).to.deep.equal({ [attestationLink]: { 'agree': { [ssid2.did]: [ { [claimlink3]: { 'need': 'water' } } ] } } })
     })
+
+    it('should be able to import multiple verifiable claims in multiple channels in order (in ephemeral connector)', async () => {
+      let ssid = await discipl.newSsid('ephemeral')
+      let ssid2 = await discipl.newSsid('ephemeral')
+
+      await discipl.claim(ssid, { 'need': 'food' })
```

Comment:
Do we need tests with respect to links that are being followed with attests? I'm not sure whether this produces special cases.

Reply:
No, links can just be imported as object of the claim as attestation. They can point to anywhere and this does not have to be checked at import.